src/app/register-email-form/register-email-form.component.html

```diff
@@ -14,13 +14,16 @@ <h1>{{MESSAGE_PREFIX + '.header'|translate}}</h1>
                     <label class="font-weight-bold"
                            for="email">{{MESSAGE_PREFIX + '.email' | translate}}</label>
                     <input [className]="(email.invalid) && (email.dirty || email.touched) ? 'form-control is-invalid' :'form-control'"
-                           type="text" id="email" formControlName="email"/>
+                           type="text" id="email" formControlName="email"
+                           [attr.aria-label]="'register-email.aria.label'|translate"
```

Comment:
On this line, we'd recommend changing the label to be: `MESSAGE_PREFIX + '.aria.label' | translate` because the `MESSAGE_PREFIX` is used for other labels in this same file.  See above on line 15.